Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM openjdk:17-slim AS builder
+FROM --platform=$BUILDPLATFORM openjdk:17-slim AS builder
```

Comment:
the `builder` step produces class files which are platform agnostic.  Therefore only need to do this step on the build platform architecture